jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/ZipManifest.kt

```diff
@@ -9,5 +9,6 @@ import software.aws.toolkits.jetbrains.services.codemodernizer.ZIP_SOURCES_PATH
 data class ZipManifest(
     val sourcesRoot: String = ZIP_SOURCES_PATH,
     val dependenciesRoot: String? = null,
-    val version: String = UPLOAD_ZIP_MANIFEST_VERSION.toString()
+    val version: String = UPLOAD_ZIP_MANIFEST_VERSION.toString(),
+    val supportPatching: Boolean = true
```

Comment:
This is an API level change to fix a customer facing issue. Will require a release outside the normal release cycle.

Reply:
Did we verify this fix has worked E2E from backend to IDE?

The expected behavior should from adding "supportPatching" should be to not add source code to the zip file on return, reducing download size. This change is specific to IntelliJ because it supports the "patch file" diff viewer and source code is not needed.